Nodejs/Product/Profiling/Profiling/StandaloneTargetView.cs

```diff
@@ -152,7 +152,7 @@ public StandaloneTargetView(StandaloneTarget template)
                     WorkingDirectory.IndexOfAny(Path.GetInvalidPathChars()) == -1 &&
                     (Path.IsPathRooted(ScriptPath) || Path.IsPathRooted(WorkingDirectory)) &&
                     File.Exists(Path.Combine(WorkingDirectory, ScriptPath)) &&
-                    (string.IsNullOrEmpty(WorkingDirectory) || Directory.Exists(WorkingDirectory)) &&
+                    (WorkingDirectory == string.Empty || Directory.Exists(WorkingDirectory)) &&
```

Comment:
This changes the behavior and some analyzers will complain about comparing to `string.Empty` directly. Is there a specific reason for this change?